top-level.nix

```diff
@@ -441,13 +443,11 @@ final: prev: with final; {
     withKerberos = true;
   };
 
-  openssl = openssl_3;
-  inherit (callPackages ./pkgs/openssl { })
-    openssl_1_1
-    openssl_3
-    openssl_3_2
-    openssl_3_3
-    ;
+  # Aliases for backwards compat
+  openssl_1_1 = openssl.v1_1;
```

Comment:
backwards with what? nixpkgs?

Reply:
yea, the hope is that coming from nixpkgs won't be as painful

similar to nixpkgs, `aliases.nix`, I was going to have a `nixpkgs-aliases.nix` lol